pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -989,7 +1002,7 @@ func (self *awsInstance) describeInstance() (*ec2.Instance, error) {
 // Gets the mountDevice already assigned to the volume, or assigns an unused mountDevice.
 // If the volume is already assigned, this will return the existing mountDevice with alreadyAttached=true.
 // Otherwise the mountDevice is assigned by finding the first available mountDevice, and it is returned with alreadyAttached=false.
-func (self *awsInstance) getMountDevice(volumeID string, assign bool) (assigned mountDevice, alreadyAttached bool, err error) {
+func (self *awsInstance) getMountDevice(volumeID string, assign bool, instanceName string) (assigned mountDevice, alreadyAttached bool, err error) {
```

Comment:
You don't need to pass in instanceName, because this is a member function of awsInstance.  So you can just get it by `self.nodeName`